frontend/src/console/actions.ts

```diff
@@ -11,6 +11,7 @@ export enum Types {
   // Support action to update root state and clean error messages when users change fields
   CLEAR_ERROR_MESSAGE = 'CLEAR_ERROR_MESSAGE',
   // To handle multiple user instances
+  USER_REFRESH_DATA = 'USER_REFRESH_DATA',
```

Comment:
This does not match what the type is used for.
Please rename it to `CLEAR_CONSOLE_DATA` or something along those lines.

The same goes for the action (`UserRefreshData` -> `ClearConsoleData`).

Reply:
Thanks for the suggestions, I have addressed it and refactored the code :)